CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ add_library(tcmu
   tcmuhandler-generated.c
```

Comment:
Could you rebuild the patch against https://github.com/open-iscsi/tcmu-runner/pull/106 when you resub
mit?